dvc/repo/ls.py

```diff
@@ -0,0 +1,107 @@
+import os
+
+from dvc.exceptions import DvcException, NotDvcRepoError, OutputNotFoundError
+from dvc.scm.base import CloneError
+
+
+@staticmethod
+def ls(url, target=None, rev=None, recursive=None, outputs_only=False):
+    if os.path.exists(url):
+        if rev:
+            raise DvcException("Local repo doesn't support revisions")
```

Comment:
There is an inconsistency with `dvc get` and `dvc import` here, as they both just `external_repo` even local urls, while `ls`treats it more like `get_url` from `api`. I do see the point in that, but it feels like we should be consistent here and since get/import are out of focus for this PR, we should make `ls` behave the same way. CC @shcheklein @Suor

Reply:
with `external_repo` `dvc list` now works similar to `dvc get`